src/migrate.js

```diff
@@ -88,6 +88,11 @@ function Migrate() {
 		}));
 	};
 
+	this.createWebhook = async function(secret, events, payloadUrl, orgName, repoName) {
```

Comment:
Make this capable of accepting multiple repos?
And is that case, we can pass repoName, secret, payloadUrl & events as webhookConfig object

Reply:
done